ethcore/types/src/errors/engine_error.rs

```diff
@@ -35,8 +35,16 @@ pub enum EngineError {
 	BadSealFieldSize(OutOfBounds<usize>),
 	/// Validation proof insufficient.
 	InsufficientProof(String),
+	/// Randomness error in load method
+	RandomnessLoadError(String),
```

Comment:
Seems that the AuRa internal randomness contract is leaking to a generic `EngineError` type, I think we should rather keep these under some more generic variant, especially that we never really handle them explicitly.

Reply:
Yes, I'm really unsure about how we should organize these errors (also see https://github.com/paritytech/parity-ethereum/pull/10946#discussion_r346882321).

Should we call this `Other(String)` then? Or `Other<Box<dyn Error>>`?

Reply:
Yeah, either of those. I think I'm more in favor of simpler `Other(String)`. Ideally would be if `Engine` had an associated type of it's custom error type, but that's way beyond the scope of this PR.

@dvdplm are you fine with `Other(String)` or do you like `Other<Box<dyn Error>>` more?

Reply:
Actually, there's already a `Custom(String)`.

Reply:
Let's go with the existing `Custom(String)`. Not ideal, but I agree with Tomek here: improving the error handling here is non-trivial and not in scope for this PR.

Reply:
Great! I used `Custom` in https://github.com/paritytech/parity-ethereum/pull/10946/commits/ab169e24b8a52539b4edef619517b347d99e4dfb.